Source/items.h

```diff
@@ -357,6 +357,16 @@ struct Item {
 		}
 	}
 
+	[[nodiscard]] bool IsScroll() const
+	{
+		return _iMiscId == IMISC_SCROLL || _iMiscId == IMISC_SCROLLT;
+	}
+
+	[[nodiscard]] bool IsScrollOf(spell_id spell) const

```

Comment:
Again would recommend renaming to `spellId`.

Reply:
Done